servers/rendering/renderer_canvas_cull.h

```diff
@@ -53,6 +53,7 @@ class RendererCanvasCull {
 		Transform2D ysort_xform;
 		Vector2 ysort_pos;
 		int ysort_index;
+		int ysort_p_abs_z_index; // absolute z index of parent. Only populated and used when ysorting.
```

Comment:
```suggestion
		int ysort_p_abs_z_index; // Absolute Z index of parent. Only populated and used when Y-sorting.
```

Does `p` here stand for "parent"? Might be worth spelling out in full as we usually use `p_` prefix for parameters, and in some of the other code you have e.g. `orig_p_z = p_z` and it's not clear whether you mean it as "original value of the `p_z` parameter" or "original value of parent Z index". If the former, we usually don't repeat the `p_` in local variables, it's meant specifically to indicate that the identifier is that of a method parameter.

Reply:
Woops, fixed the naming. I assumed p_ stood for parent in the original code.